servicepulse/intro-editing-messages.md

```diff
@@ -11,7 +11,25 @@ If a message cannot be successfully retried it is possible to fix the malformed
 
 ## Enabling the feature
 
-The message edit and retry feature is not enabled by default. To enable it, contact [Particular Support](https://particular.net/support) for assistance.
+{{DANGER:
+Editing malformed messages is a potentially destructive operation, which if used improperly can have consequences that affect the correctness of the system. If a malformed message is edited improperly, it could still be processed successfully, without any way to undo the effects. The possible effects of editing a malformed message are system-specific and ServicePulse is unable to determine if a potential edit operation is safe for the system. There is also no way to limit failed message editing only to certain security groups or to specific message types.
```

Comment:
Should we add "at this time"? Or will it take so long to implement, that it'll probably be 2025 before it'll work? What do you think?
```suggestion
Editing malformed messages is a potentially destructive operation, which if used improperly can have consequences that affect the correctness of the system. If a malformed message is edited improperly, it could still be processed successfully, without any way to undo the effects. The possible effects of editing a malformed message are system-specific and ServicePulse is unable to determine if a potential edit operation is safe for the system. At this moment there is also no way to limit failed message editing only to certain security groups or to specific message types.
```